src/integrationTest/java/uk/gov/hmcts/darts/testutils/stubs/ArmServiceStubImpl.java

```diff
@@ -19,6 +23,45 @@ public String saveBlobData(String containerName, String filename, BinaryData bin
         return filename;
     }
 
+    @Override
+    public String saveBlobData(String containerName, BinaryData binaryData, String blobPathAndName) {
+        logStubUsageWarning();
+        log.warn("Returning filename to mimic successful upload: {}", blobPathAndName);
```

Comment:
Returning blobPathAndName?

Reply:
ARM does not save to a UUID but to the blob path and name so that is what is returned